sys/net/crosslayer/ng_netreg/ng_netreg.c

```diff
@@ -20,7 +20,19 @@
 #include "net/ng_netreg.h"
 #include "net/ng_nettype.h"
 #include "net/ng_pkt.h"
+
+#ifdef MODULE_NG_IPV6
 #include "net/ng_ipv6.h"
+#endif
+#ifdef MODULE_NG_ICMPV6
+#include "net/ng_icmpv6.h"
```

Comment:
Is ICMPv6 really needed here? It will not serve the header building function, since there are no ICMPv6 headers, just packets ;-).